### ISSUE TITLE
Getting Started with Blockly, Step 7 breaks with additions to main.js

### ISSUE THREAD

**Issue by sabrams61** [rachel-fenichel/blockly-samples#1118]
I made the following edits to blockly-samples/examples/getting-started-codelab/starter-code/scripts/main.js:

```
function save(button) {
      button.blocklyXml = Blockly.Xml.workspaceToDom(Blockly.getMainWorkspace()); // added this line
}
```

```
function loadWorkspace(button) { // added this function
  let workspace = Blockly.getMainWorkspace();
  workspace.clear();
  if (button.blocklyXml) {
    Blockly.Xml.domToWorkspace(button.blocklyXml, workspace);
  }
}
```

```
function enableBlocklyMode(e) {
    document.body.setAttribute('mode', 'blockly');
    currentButton = e.target;
    loadWorkspace(currentButton); // added this line
  }
```

And when I refreshed the page and clicked "Edit" and then one of buttons, the page broke and I got this console error:

```
main.js:16 Uncaught TypeError: Cannot read properties of undefined (reading 'clear')
    at loadWorkspace (main.js:16:15)
    at HTMLDivElement.enableBlocklyMode (main.js:50:5)
```

**Comment by gonfunko** [comment]
Thanks for reporting this, I'm unfortunately not able to reproduce it. To double check, was/is this still happening after sorting out the load order in #1117? Additionally, which browser are you using?

**Comment by BeksOmega** [comment]
Closed by #1522
